recipe/data.go

```diff
@@ -24,7 +24,7 @@ var CPI = Response{
 	},
 	OutputInstances: []instance{
 		{
-			DatasetID: "931a8a2a-0dc8-42b6-a884-7b6054ed3b68",
+			DatasetID: "cpi",
```

Comment:
Probably want to leave this one as an exception to the lower case rule as used for testing and devs running locally.

Reply:
Good point. Have reapplied the old code with a commit.